microsoft-edge/webview2/release-notes.md

```diff
@@ -31,6 +31,113 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 To use a prerelease SDK along with a Microsoft Edge preview channel, navigate to [Switch to a preview channel to test upcoming APIs and features](how-to/set-preview-channel.md).
 
 <!-- ====================================================================== -->
+
+## 1.0.1071.54
+  
+Release Date: January 10, 2022  
+  
+[NuGet package for WebView2 SDK 1.0.1071.54](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1071.54-release)  
+  
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 97.1071.54 or higher.  
+
+### General
+
+#### Promotions
+  
+The following items are now stable:
+*  The [Media API](/microsoft-edge/webview2/reference/win32/icorewebview2experimental9?view=webview2-1.0.1083-prerelease&preserve-view=true#summary) that enables developers to mute/unmute media within the WebView.
+*  The [Download Positioning and Anchoring API](/microsoft-edge/webview2/reference/win32/icorewebview2experimental11?view=webview2-1.0.1083-prerelease&preserve-view=true).  This API enables:
+   *  Changing the position of the download dialog, relative to the WebView2 bounds.  You can anchor the download dialog to the **Download** button, instead of the default position, which is the top-right corner.
+   *  Programmatically opening and closing the default download dialog.
+   *  Making changes in response to the dialog opening and closing.
+
+//Ask @Jessica
+API promotions/removals in 
+D:\\Edge\\src\\edge_embedded_browser\\client\\win\\current\\webview2experimental.idl:
+
+ICoreWebView2ExperimentalFrame2
+ 
+(ADDED)
+     PostWebMessageAsJson
+     PostWebMessageAsString
+     add_WebMessageReceived
+     remove_WebMessageReceived
+ICoreWebView2ExperimentalFrameWebMessageReceivedEventHandler
+ (ADDED)
```

Comment:
@peiche-jessica - this experimental idl diff showed up in the release section of the script. Not sure why